### PR TITLE
Fix two term bug and styling changes

### PR DESCRIPTION
**Pull request by mlool**
### What Issue does this PR resolve? (Link to GitHub Issue, approved features and bugs will be given priority)

1. Styling change
2. Fixed 2 term conflict bug
3. Fixed import doesn't fully fetch data

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "UBC Workday Side by Side Calendar",
-  "version": "1.5.1",
+  "version": "1.6.0",
```
Comment (ThatNerdSquared) [inline public/manifest.json:4]: can you update the firefox manifest as well?

File: src/components/TabBar.tsx [hunk 2]
```diff
@@ -7,13 +7,16 @@ interface TabBarProps<T> {
...
   tabTextBuilder?: (item: T) => string
+  focus: boolean
 }
 
 export default function TabBar<T>(props: TabBarProps<T>) {
   const determineTabColor = (item: T) => {
     if (props.isSelected(item)) return "tab-selected"
     if (props.isHighlighted && props.isHighlighted(item))
       return "tab-highlighted"
+    if (props.focus && !props.isSelected(item))
```
Comment (ThatNerdSquared) [inline src/components/TabBar.tsx:18]: second condition here is unnecessary, if it were true the function would have already returned by this point.

also, can we rename `props.focus` to something more descriptive, like `props.greyOutBackgrounded` or something lol

Comment (mlool) [reply src/components/TabBar.tsx:18]: Updated

File: src/content/Calendar/calendarHelpers.ts [hunk 3]
```diff
@@ -139,8 +138,7 @@ export const convertToMatrix = (
     })
   })
 
-  setSectionConflict(hasInvalidSection)
-  return matrixDict
+  return [hasInvalidSection, matrixDict]
```
[the remark this answers is not in the document]
Comment (mlool) [reply src/content/Calendar/calendarHelpers.ts:141]: This is a tuple, so should be fine as it's very much enforced

File: public/manifest.json (hunk 1 above)
Comment (mlool) [reply public/manifest.json:4]: sure

Comment (mlool) [reply public/manifest.json:4]: wait where is the firefox manifest?

**Pull request merged**